src/server/common.h

```diff
@@ -377,6 +377,30 @@ class ABSL_LOCKABLE ThreadLocalMutex {
   util::fb2::detail::FiberInterface* locked_fiber_{nullptr};
 };
 
+// Replacement of std::SharedLock that allows -Wthread-safety
+template <typename Mutex> class ABSL_SCOPED_LOCKABLE SharedLock {
```

Comment:
but we'll follow up with moving that to helio, right?

Reply:
If Roman doesn't have any issues yes